Nexus/Include/Nexus/FeeHandlingTests/FeeTableTestUtilities.hpp

```diff
@@ -108,7 +108,7 @@ namespace Nexus::Tests {
   template<typename FeeTable, typename CalculateFeeType>
   void TestNotionalValueFeeCalculation(const FeeTable& feeTable,
       const OrderExecutionService::OrderFields& orderFields,
-      CalculateFeeType&& calculateFee, Money expectedRate) {
+      CalculateFeeType&& calculateFee, Quantity expectedRate) {
```

Comment:
I am not sure how it worked before as Money was multiplied by Money on line 117. It compiles after making it Quantity.